client/src/modules/Registry/components/Footer/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Image, Space, Typography } from 'antd';
+import css from 'styled-jsx/css';
+
+const copyrights = `Copyright © The Rolling Scopes ${new Date().getFullYear()}`;
+const logos = [
+  {
+    src: '/static/svg/logo-rs.svg',
+    alt: 'Rolling Scopes Logo',
+  },
+  {
+    src: '/static/svg/logo-github.svg',
+    alt: 'GitHub Logo',
+  },
+  {
+    src: '/static/svg/logo-epam.svg',
+    alt: 'EPAM Logo',
+  },
+];
+
+export function Footer() {
+  return (
+    <>
+      <footer>
+        <Space align="center" size={40} style={{ height: '32px' }}>
+          {logos.map(({ src, alt }) => (
+            <Image key={alt} preview={false} src={src} alt={alt} />
```

Comment:
Please consider to use Custom Icon https://ant.design/components/icon#custom-svg-icon

Reply:
fixed